src/scripts/ordo_report_mapping_annotations/create_sparql__ordo_add_annotation_based_mappings.py

```diff
@@ -0,0 +1,135 @@
+"""Update ORDO mapping annotations to formalize them semantically.
+Work described furthere here: https://github.com/monarch-initiative/mondo-ingest/issues/23
+
+Resources
+1. https://www.orpha.net/orphacom/cahiers/docs/GB/Orphanet_ICD10_coding_rules.pdf
+
+Prerequisites to run this script:
+1. `src/ontology/mirror/ordo.owl` must be present. To get it, run `sh src/ontology/run.sh make refresh-ordo`.
```

Comment:
this is not correct anymore, 

```
sh run.sh make component-download-ordo.owl
```

which creates:

```
src/ontology/tmp/ordo.owl.owl
```

(.owl twice, for some technical convenience.).

Reply:
@matentzn Sounds good. However, perhaps this is a bug. I just did a `git pull` and ran the command, but it created `tmp/component-download-ordo.owl.owl` instead of `tmp/ordo.owl.owl`.